app/helpers/pagination_helper.rb

```diff
@@ -0,0 +1,40 @@
+##
+# This contains methods to help calculate and represent pagination data.
+module PaginationHelper
```

Comment:
I added this to account for cases when there may be a very large amount of results in the response. It's pretty standard for me when I implement APIs that return a list of things. A smaller amount of results does lower response time and saves on space/time when working on a smaller result set. But this is mostly for establishing structure and handling common pagination functionality. I hadn't actually looked into existing libraries since I actually implemented this in Go last year at RepIQ and it was heavily used. I wanted to use the same logic myself without having to install another library.